src/Castle.Core.Tests.WeakNamed/Castle.Core.Tests.WeakNamed.csproj

```diff
@@ -16,6 +16,7 @@
 		<SignAssembly>False</SignAssembly>
 		<PublicSign Condition="'$(OS)'=='Unix'">false</PublicSign>
 		<StartupObject>Program</StartupObject>
+		<InvariantGlobalization>true</InvariantGlobalization>
```

Comment:
Is this change strictly required for offloading `DiagnosticsLogger` into a separate project/package? If not (which I suspect), please undo this change.